templates/account/account.py

```diff
@@ -1,28 +1,30 @@
 from js9 import j
 from zerorobot.template.base import TemplateBase
+from zerorobot.template.state import StateCheckError
 
 
 class Account(TemplateBase):
 
     version = '0.0.1'
     template_name = "account"
 
+    OVC_TEMPLATE = 'github.com/openvcloud/0-templates/openvcloud/0.0.1'
+    VDCUSER_TEMPLATE = 'github.com/openvcloud/0-templates/vdcuser/0.0.1'
+
     def __init__(self, name, guid=None, data=None):
         super().__init__(name=name, guid=guid, data=data)
 
     def validate(self):
-        OVC_TEMPLATE = 'github.com/openvcloud/0-templates/openvcloud/0.0.1'
-        ovcs = self.api.services.find(template_uid=OVC_TEMPLATE, name=self.data.get('openvcloud', None))
+        ovcs = self.api.services.find(template_uid=self.OVC_TEMPLATE, name=self.data['openvcloud'] or None)
```

Comment:
Why the `or None` for the name argument. openvcloud connection should always be specified no ?

Reply:
No, it's actually optional (in case u only have one openvcloud connection) in this case it makes your bp easier to write down. if it's not specified, the `self.data['openvcloud']` will actually return an empty string, hence i do call the find with `None` to make sure it's the default find behaviour with a template_uid only.

Later on, once the connection is found, it's stored in the data.

Reply:
is it something we really want to do ? I find that a bit error prone. Why would you rely on the fact that could have only one. It feels like automagic